security/compliance_operator/compliance-operator-crd.adoc

```diff
@@ -0,0 +1,494 @@
+:_content-type: ASSEMBLY
+[id="custom-resource-definitions"]
+= Understanding the Custom Resource Definitions
+include::_attributes/common-attributes.adoc[]
+:context: custom-resource-definitions
+
+toc::[]
+
+The Compliance Operator in the {product-title} provides you with several Custom Resource Definitions (CRDs) to accomplish the compliance scans. To run a compliance scan, it leverages the predefined security policies, which are derived from the link:https://github.com/ComplianceAsCode/content[ComplianceAsCode] community project. The Compliance Operator enables these security policies into CRDs, which you can use to run compliance scans and get remediations for the issues found.
+
+[id="custom-resource-definitions-workflow_{context}"]
+== CRDs workflow
+
+The CRD provides you the following workflow to complete the compliance scans:
+
+. Define your compliance scan requirements
+. Configure the compliance scan settings
+. Process compliance requirements with compliance scans settings
+. Monitor the compliance scans
+. Check the compliance scan results
+
+[id="defining-compliance-scan-requirements_{context}"]
+== Defining the compliance scan requirements
+By default, the Compliance Operator CRDs include `ProfileBundle` and `Profile` objects, in which you can define and set the rules for your compliance scan requirements. You can also customize the default profiles by using a `TailoredProfile` object.
+
+[id="profile-bundle-object_{context}"]
+=== ProfileBundle object
+You can create `ProfileBundle` object to make it easier for users to determine which profiles are included in a container. The Compliance Operator parses the `ProfileBundle` object and creates a `Profile` object for each profile in the bundle. It also parses `Rule` and `Variable` objects, which are used by the `Profile` object.
```

Comment:
Ah, I see now, the sentence was taken from the upstream CRD doc. I think it makes sense upstream, because there you can expect users to play around with bundles more frequently, but in OCP downstream, I think people will be just directly consuming existing bundles.

Reply:
Fixed.